pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features.adoc

```diff
@@ -160,3 +144,22 @@ async fn configure(launcher: Launcher, Configuration(bytes): Configuration) -> R
     Ok(())
 }
 ----
+
+[[provided-rust-source-code]]
+== Provided Rust Source Code
```

Comment:
I would rename this title, to also express that this functionality is exposed by our rust libraries, like:

"Discovering the PDK features with provided source code"